src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -36,6 +39,11 @@ uct_dc_mlx5_set_dgram_seg(uct_ib_mlx5_txwq_t *txwq,
     to_av->fl_mlid      = iface->tx.av_fl_mlid;
     to_av->rlid         = av->rlid;
 
+    /* Setting reverse_sl */
+    to_av->dqp_dct &= ~(REVERSE_SL_MASK);
+    to_av->dqp_dct |= (iface->super.super.super.config.reverse_sl &
+                       REVERSE_SL_MASK);
```

Comment:
extra ( ) in both lines

Reply:
Fixing in: https://github.com/openucx/ucx/pull/9510